vast/vast3/wrapper.go

```diff
@@ -0,0 +1,44 @@
+package vast3
+
+import vastbasic "github.com/Vungle/vungo/vast/basic"
+
+// Wrapper type represents a <Wrapper> element in VAST document that contains a single URI
+// reference to a vendor ad server, a.k.a. third party ad server, where the actual VAST w
+// content will be served. The vendor ad server may also provide additional wrapper which
+// eventually resolves to the actual ad.
+type Wrapper struct {
+	AdSystem     *vastbasic.AdSystem     `xml:"AdSystem"`     // Required.
+	VastAdTagURI string                  `xml:"VASTAdTagURI"` // Required.
```

Comment:
should define it as trimData

Reply:
Change to TrimmedData to enable the ability to trim space for URI.